tests/regression/lp-1644439/kernel-new-enough.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+# Check if the kernel is at least 4.4.0-67
+if ! uname -r | perl -ne '/^(\d+)\.(\d+)\.(\d+)-(\d+)/ or exit 1; exit 1 if $1<4; exit 1 if $2<4; exit 1 if $3==0 && $4<67'; then
```

Comment:
Let's please skip if not in Ubuntu as well. We know for sure this makes no sense anywhere else but Ubuntu, and we know we want to test other distributions already.